lib/scrabble_master.rb

```diff
@@ -1,2 +1,7 @@
 require_relative "scrabble"
 require_relative "player"
+require_relative "tilebag"
+require_relative "dictionary"
+require_relative "game"
+
+play_scrabble
```

Comment:
It's not a good idea to place a method call to `play_scrabble` here, as every time a file `require`s anything from the Scrabble module, it will start playing the game.

For example, after the commit that introduced this (0c12df) it is not possible to run `rspec` on your project without starting the game (which doesn't seem to have an end?)